### ISSUE TITLE
Improvements to fix the accessibility of the app

### ISSUE THREAD

**Issue by anmi2004** [seerge/g-helper#1823]
## Problem Description

While using a screen reader, when browsing the app I can only view the description of the controls in English language.Even though my app is in Chinese
Some of the controls on the panel are not interactable, such as the slider that controls the charging threshold

## Ideal Performance

When interacting with the application, the text broadcast by the screen reader should be in the same language as what is actually being displayed
Both buttons and sliders should be able to interact properly and announce their status properly.

## Additional Information

Windows 11 23H2 (AMD64) build 22631.2861
Device, Chosen Four
Test screen readers are Microsoft Narrator and [nvda](https://github.com/nvaccess/nvda)

**Comment by anmi2004** [comment]
I'm blind and I'm fed up with all the problems caused by this Asus crapware.So sent this topic in the hope that the developers will improve it.Because I use a translator to communicate, some parts may not be expressed accurately enough.Your understanding is appreciated, thank you.

**Comment by seerge** [comment]
@anmi2004 hello, this build should have translate Accessible Labels (ones that narrator reads)

[GHelper.zip](https://github.com/seerge/g-helper/files/13801414/GHelper.zip)

**Comment by seerge** [comment]
P.S. Please keep in mind, that some sections of the app are increadibly difficult to make completely accessible. For example fan curve editor, that is simply drag-n-drop only :) 

But main buttons / labels that I was intentionally making accessible by request of other blind person - should have translations.

Good luck :)

**Comment by anmi2004** [comment]
# There are a couple of other minor issues that I'm not sure can be optimized

- If I select Enhanced Mode or Silent Mode, can I broadcast the selected state when navigating to the control.
- When interacting with the lock fn key feature, I can't read if the state is locked or not.
- I was unable to determine if the Charge Limit feature was enabled or not, and the progress bar for that feature was not interacting properly.

Thank you very much for your prompt help, I didn't expect to solve this problem so quickly, it's much appreciated!
